data/story_types.json

```diff
@@ -1,198 +1,477 @@
-    "key_moment": [
-      "Revelation that untangles the confusion and restores harmony.",
-      "Group recognition of mistaken assumptions.",
-      "A union or celebration that resets social balance."
-    ],
-    "core_arc": [
-      "Disorder unraveled through wit.",
-      "Authenticity replaces pretense.",
-      "Misunderstandings resolved through humility."
-    ],
-    "common_elements": [
-      "Mistaken assumptions",
-      "Disguises",
-      "Romantic entanglements",
-      "Festive ending"
-    ],
-    "selectable_fields": ["key_theme", "key_moment", "core_arc"]
-  },
-  {
-    "plot": "Tragedy",
-    "description": "A protagonist with a fatal flaw experiences rise and inevitable downfall, provoking audience catharsis.",
-    "narrative_rhythm": "Ambition, Success, Decline, Crisis, Collapse",
-    "key_theme": [
-      "Human flaws and hubris lead to irreversible consequences.",
-      "The more one rises, the farther there is to fall.",
-      "Blindness to truth is the seed of destruction."
-    ],
-    "emotional_arc": ["Pride", "Shame", "Despair", "Catharsis"],
-    "key_moment": [
-      "Collapse or loss that exposes the protagonist’s tragic flaw.",
-      "An irreversible choice seals their fate.",
-      "Moment of self-awareness that comes too late."
-    ],
-    "core_arc": [
-      "Hero’s virtue becomes vice.",
-      "Blind ambition leads to ruin.",
-      "Fate and flaw converge in downfall."
-    ],
-    "common_elements": [
-      "Fatal flaw",
-      "Tempting success",
-      "Betrayal",
-      "Irreversible consequence"
-    ],
-    "selectable_fields": ["key_theme", "key_moment", "core_arc"]
-  },
-  {
-    "plot": "Rebirth",
-    "description": "Hero experiences a symbolic death or spiritual crisis and is restored through transformation.",
-    "narrative_rhythm": "Stagnation, Fall, Awakening, Redemption, Renewal",
-    "key_theme": [
-      "Even the lost can rediscover purpose and hope.",
-      "Compassion can reignite the human spirit.",
-      "Transformation begins with self-awareness."
-    ],
-    "emotional_arc": ["Apathy", "Darkness", "Epiphany", "Rejuvenation"],
-    "key_moment": [
-      "Moment of awakening that breaks isolation.",
-      "Act of kindness that inspires renewal.",
-      "Epiphany during confrontation with past mistakes."
-    ],
-    "core_arc": [
-      "From brokenness to restored identity.",
-      "Redemption through empathy.",
-      "Transformation through internal clarity."
-    ],
-    "common_elements": [
-      "Moral decay",
-      "Ghost of former self",
-      "Outsider intervention",
-      "Emergence with new outlook"
-    ],
-    "selectable_fields": ["key_theme", "key_moment", "core_arc"]
-  }
-]
+{
+  "story_types": [
+    {
+      "name": "Overcoming the Monster",
+      "description": "Hero faces a great evil or threat",
+      "examples": [
+        "Beowulf",
+        "Jaws"
+      ],
+      "characteristics": {
```

Comment:
get rid of the "characteristics" data.  This was made obsolete by the new merged data